chapters/daemon/index.rst

```diff
@@ -0,0 +1,6 @@
+Jina Daemon API Specification
+=============================
+
+Link to Jina Daemon OpenAPI Specification <https://api.jina.ai/daemon/>
```

Comment:
Simple and clean! I like it. 

One comment, can you convert this line (and line 4 in the other file) to active voice please? 

_Use this link to find the Jina Daemon OpenAPI Specification_